### DISCUSSION TITLE
Long clips options

### DISCUSSION THREAD

**Discussion by cliffm** [blakeblackshear/frigate#876] [Ideas]
I am using the clips feature, and I came across this in the documentation:

```
# Optional: Maximum length of time to retain video during long events. (default: shown below)
# NOTE: If an object is being tracked for longer than this amount of time, the cache
#       will begin to expire and the resulting clip will be the last x seconds of the event.
max_seconds: 300
```

Is there a way to make it do the opposite? Basically trigger a clip recording at the start, and then cut it off when it reaches max? Instead of recording the entire "event", and only saving the last few seconds/minutes?

**Comment by blakeblackshear** [comment]
Not currently. It doesn’t work like you described. It just assembles a clip from the cache when the event ends. The cache from the start is no longer there because it has expired.
